### PR TITLE
Update gce compute api godep

### PR DESCRIPTION
**Pull request by bprashanth**
Essentially 

```
godep restore ./... 
go get -u google.golang.org/api/compute/v1/
godep update google.golang.org/api/...
```

Needed for GCE Https resources. 
@kubernetes/goog-gke for review?

### CODE REVIEW

**Comment by bprashanth** [comment]
Pseudo random assignment to CJ, feel free to re-assign

File: pkg/cloudprovider/providers/gce/gce.go [hunk 1]
```diff
@@ -1367,12 +1367,14 @@ func (gce *GCECloud) AddSSHKeyToAllInstances(user string, keyData []byte) error
 		found := false
 		for _, item := range project.CommonInstanceMetadata.Items {
 			if item.Key == "sshKeys" {
-				if strings.Contains(item.Value, keyString) {
+				value := *item.Value
+				if strings.Contains(value, keyString) {
```
Comment (cjcullen) [inline pkg/cloudprovider/providers/gce/gce.go]: I'd write this as 
```
if strings.Contains(*item.Value, keyString) {...}
```
and only construct a new string if necessary below:
```
value := *item.Value + "\n" + keyString
```

**Comment by cjcullen** [comment]
Large dependency changes are a little intimidating, but I trust that this should be effectively no difference. Small amount of code LGTM (modulo minor nit), and godep updates look reasonable.

**Comment by bprashanth** [comment]
Done. Hoping our tests will catch any breakage. I've only verified that Godeps.json looks reasonable and that the new godep contains the changes I need.

**Comment by bprashanth** [comment]
@cjcullen trap is green

**Comment by cjcullen** [comment]
Sorry. Thought I'd LGTM'd this.

**Comment by bprashanth** [comment]
```
NAME                           NETWORK             SRC_RANGES    RULES                    SRC_TAGS TARGET_TAGS
e2e-gce-builder-1-7-minion-all e2e-gce-builder-1-7 10.245.0.0/16 tcp,udp,icmp,esp,ah,sctp          e2e-gce-builder-1-7-minion
ERROR: (gcloud.compute.instances.create) Some requests did not succeed:
 - Quota 'IN_USE_ADDRESSES' exceeded.  Limit: 65.0
 
```

**Comment by bprashanth** [comment]
@k8s-bot test this please

**Pull request merged**